### ISSUE TITLE
CVODE Parallel 

### ISSUE THREAD

**Issue by ivanZanardi** [modelon-community/Assimulo#67]
Hi, thanks for this nice package.

Just wondering if it is possible to run CVODE with multiple threads with DENSE linear solver.

Thanks.
Ivan

**Comment by PeterMeisrimelModelon** [comment]
Hi Ivan,

Multithreaded LU factorization in CVode is currently only available when using Sparse matrices & SuperLU.

From a brief look into CVode's linear solver module, the only type of threading available for dense linear solves would be via using NVector_pthreads, i.e., a threaded vector implementation, which may only have a marginal effect, if any, on speeding up the LU factorization. 

I may be missing something here though, I didn't check all linear solver modules available in Sundials.

/Peter

**Comment by ivanZanardi** [comment]
Hello, thank you for your response.
I appreciate your explanation.

Let me rephrase my question: Is it possible to use the dense linear solver in conjunction with OpenBLAS-linked LAPACK for multithreaded operation?

I apologize for any confusion in my previous question.
Thank you for your time.

**Comment by PeterMeisrimelModelon** [comment]
Hi Ivan,

CVode with the dense direct linear solver currently only supports the inbuilt LU factorization routines, not LAPACK/BLAS.

/Peter
